src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,53 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import core.basesyntax.exeptions.StorageException;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAXIMUM_STORAGE_LENGTH = 10;
+    private int lastItemIndex = 0;
```

Comment:
ints are 0 by default
```suggestion
    private int lastItemIndex;
```